### PR TITLE
Add tabs with positions and interviews list to the UI

### PR DESCRIPTION
**Pull request by likeabowx**
- Added tabs to allow display of different type of data
- Added cards and lists to display positions and interviews
- Currently using hard coded data to display positions and interviews, need to replace with the ObservableList from logic whenever ready @goalfix @YiHern-Lee 

I'm open to any feedback and suggestion for changes to the design, do let me know.

This is how the positions tab looks like now:
image

And this is the interviews tab:
image

### CODE REVIEW

File: src/main/java/seedu/address/ui/InterviewCard.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    /**
+     * Creates a {@code InterviewCard} with the given {@code Interview} and index to display.
+     */
+    public InterviewCard(Interview interview, int displayedIndex) {
+        super(FXML);
+        this.interview = interview;
+
+        id.setText(displayedIndex + ". ");
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("EEEE dd MMM yyyy, HH:mma");
```
Comment (YiHern-Lee) [inline src/main/java/seedu/address/ui/InterviewCard.java:53]: Would it be better if `DateTimeFormatter formatter` is a static object?
E.g. 
```java
private static final DateTimeFormatter formatted = DateTimeFormatter.ofPattern("EEEE dd MMM yyyy, HH:mma")
```
This prevents reinitialization of formatter everything the constructor is called.

**Review by YiHern-Lee — APPROVED** [review]
Overall, LGTM. Just a minor issue with regards to the constructor of `InterviewCard`.

**Comment by goalfix** [comment]
@likeabowx I'm currently done with Observable<Interviews>! Will take a look at your PR now

**Review by goalfix — APPROVED** [review]
Minor issues suggested about the date time formating for the interview card. Otherwise, good job on digesting the javaFX code and LGTM!

**Review by YiHern-Lee — APPROVED** [review]
LGTM! Agree with the scope change for `formatter` in `InterviewCard`.

File: src/main/java/seedu/address/ui/InterviewCard.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import seedu.address.model.applicant.Applicant;
+import seedu.address.model.interview.Interview;
+
+/**
+ * An UI component that displays information of a {@code Interview}.
+ */
+public class InterviewCard extends UiPart<Region> {
+
+    private static final String FXML = "InterviewListCard.fxml";
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("EEEE dd MMM yyyy, HH:mma");
```
[the remark this answers is not in the document]
Comment (likeabowx) [reply src/main/java/seedu/address/ui/InterviewCard.java:18]: I see your point, but I formatted the output in a way that I feel is easier to read, which includes the month spelt out. Personally, I think that the format `yyyy-MM-dd HH:mm` is easy for the user to type but not so friendly to read, especially when put together with many other dates in the list which will just look like a bunch of numbers.

**Review by khoahre123 — APPROVED** [review]
LGTM! Everything looks good now

**Pull request merged**